src/org/parosproxy/paros/extension/ExtensionPopupMenuItem.java

```diff
@@ -54,12 +57,43 @@
 	private boolean precedeWithSeparator = false;
 	private boolean succeedWithSeparator = false;
 
+    /**
+     * Constructs a {@code ExtensionPopupMenuItem} with no text nor icon.

```

Comment:
It's a minor detail but these should all be "an" since when rendered the next word will start with an E (vowel).